plonky2x/src/frontend/eth/mpt/builder.rs

```diff
@@ -51,7 +51,26 @@ impl<L: PlonkParameters<D>, const D: usize> CircuitBuilder<L, D> {
             _phantom: PhantomData,
         };
         self.add_simple_generator(generator.clone());
-        generator.output
+
```

Comment:
Would be great to have a bunch of test cases for this testing the edge cases and also overflow/underflow situations

Reply:
added general test in https://github.com/succinctlabs/succinctx/pull/152/commits/b79b4e7d2d4fc49e75e4aced94cddf1a78a0a9c7, need to add specific edge case tests

Reply:
added some edge case test in https://github.com/succinctlabs/succinctx/pull/152/commits/da43273874a36ba41bf2ba3d333211c7232107d2